src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
The `fruits` map is declared as `public static final`, which allows direct access and modification from outside the class. This can lead to potential data integrity issues. Consider making the map private and providing public methods to access and modify its contents.